src/low-level/smtp/mailsmtp.c

```diff
@@ -530,6 +530,9 @@ int mailsmtp_data_message_quit_no_disconnect(mailsmtp * session,
     return MAILSMTP_ERROR_STREAM;
   
   r = send_quit(session);
+  if (r == -1)
```

Comment:
Maybe use that instead:

```
if (r != MAILSMTP_NO_ERROR) {
  return r;
}
```

Reply:
Yes, can be.
But and the code before, I follow the style, fix too?
if (r != MAILSMTP_NO_ERROR) {
  return r;
}

Reply:
Done.